### DISCUSSION TITLE
Finetune on LJSpeech or LibriTTS model?

### DISCUSSION THREAD

**Discussion by Weroxig** [yl4579/StyleTTS2#225] [Q&A]
> StyleTTS 2 surpasses human recordings on the single-speaker LJSpeech dataset and matches it on the multispeaker VCTK dataset as judged by native English speakers. Moreover, when trained on the LibriTTS dataset, our model outperforms previous publicly available models for zero-shot speaker adaptation

The abstract seems to suggest that the ljspeech model is better. should I finetune using the ljspeech or libritts model?

My dataset is single speaker like:
```
1.wav|phonemes|0
2.wav|more phoneme symbols|0
```

edit: was the LJSpeech model a finetune of the LibriTTS model?

**Comment by 78Alpha** [comment]
Most people use the LibriTTS model. The LJSpeech one is better, but in my experience, I am unable to finetune it. It throws key errors, so it is likely a model all on its own and not finetuned from LibriTTS.
